packages/components/app/styles/components/dropdown.scss

```diff
@@ -278,61 +278,17 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
 
 .hds-dropdown-list-item--variant-copy-item {
   width: 100%;
+  max-width: 100%;
   padding: 10px 16px 12px;
 
-  button {
-    display: flex;
-    justify-content: space-between;
+  .hds-copy-snippet {
```

Comment:
these stiles should not live here but in the CSS code for the `Copy::Snippet` component (so the other PR); plus I think you need only the `overflow-wrap` (TBD if `anywhere` or `break-word`)

Reply:
I think they were not in that other component yet, which is why I might have felt the need to add them here. I will evaluate and address appropriately in the feature branch.